lib/db_utils.go

```diff
@@ -559,7 +559,21 @@ type DBPrefixes struct {
 	// Note, we parse the ValidatorPKID and StakerPKID from the key.
 	PrefixSnapshotStakeToRewardByValidatorAndStaker []byte `prefix_id:"[90]" is_state:"true"`
 
-	// NEXT_TAG: 91
+	// PrefixLockedBalanceEntryByHODLerPKIDProfilePKIDUnlockTimestampNanoSecs:
+	// Retrieves LockedBalanceEntries that may or may not be claimable for unlock.
+	// LockedBalanceEntries can be retrieved by HodlerPKID and CreatorPKID are have their
+	// corresponding unlock timestamp appended to sort by timestamp.
+	// Prefix, <HodlerPKID [33]byte>, <ProfilePKID [33]byte>, <UnlockTimestampNanoSecs int64> -> <LockedBalanceEntry>
+	PrefixLockedBalanceEntryByHODLerPKIDProfilePKIDUnlockTimestampNanoSecs []byte `prefix_id:"[91]" is_state:"true"`
+
+	// PrefixLockupYieldCurvePointByProfilePKIDAndDurationNanoSecs:
+	// Retrieves a LockupYieldCurvePoint.
+	// The structure of the key enables quick lookups for a (ProfilePKID, Duration) pair as well
+	// as quick construction of yield curve plots over time.
+	// Prefix, <ProfilePKID [33]byte>, <LockupDurationNanoSecs int64> -> <LockupYieldCurvePoint>
+	PrefixLockupYieldCurvePointByProfilePKIDAndDurationNanoSecs []byte `prefix_id:"[92]" is_state:"true"`
+
+	// NEXT_TAG: 93
```

Comment:
The recently merged state syncer code adds new prefixes that precede these ones. Once you merge the state syncer code into your feature branch, you'll need to update the prefix values 91 and 92 to 93 and 94 respectively. The next available tag will be 95